or-model/src/test/java/org/openrefine/preference/TopListTests.java

```diff
@@ -24,25 +24,25 @@
  * ARISING IN ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE
  * POSSIBILITY OF SUCH DAMAGE.
  ******************************************************************************/
-package com.google.refine.preference;
+package org.openrefine.preference;
 
 import java.io.IOException;
 import java.util.Collections;
 
+import org.openrefine.preference.PreferenceValue;
+import org.openrefine.preference.TopList;
+import org.openrefine.util.ParsingUtilities;
+import org.openrefine.util.TestUtils;
 import org.testng.annotations.Test;
 
 import com.fasterxml.jackson.core.JsonParseException;
 import com.fasterxml.jackson.databind.JsonMappingException;
-import com.google.refine.preference.PreferenceValue;
-import com.google.refine.preference.TopList;
-import com.google.refine.util.ParsingUtilities;
-import com.google.refine.util.TestUtils;
 
 public class TopListTests {
     @Test
     public void serializeTopList() throws JsonParseException, JsonMappingException, IOException {
         String json = "{"
-                + "\"class\":\"com.google.refine.preference.TopList\","
+                + "\"class\":\"org.openrefine.preference.TopList\","
```

Comment:
I've updated tests so that they point to an ostensibly old class name (`com.google.gridworks`) to make it clear that we're testing for backwards compatibility in https://github.com/wetneb/OpenRefine/pull/335/commits/ef5f47b53eaa1edf954fe36eccc849355fc89b70